audbackend/core/errors.py

```diff
@@ -4,14 +4,24 @@ class BackendError(Exception):
     Args:
         exception: exception raised by backend
 
+    .. Prepare backend and interface for docstring examples
+
+        >>> import audeer
+        >>> audeer.rmdir("host", "repo")
+        >>> _ = audeer.mkdir("host")
+        >>> FileSystem.create("host", "repo")
+
     Examples:
+        >>> backend = FileSystem("host", "repo")
         >>> try:
-        ...     unversioned.checksum("/does/not/exist")
+        ...     interface = audbackend.interface.Unversioned(backend)
```

Comment:
It feels a bit weird we use `audbackend.interface.Unversioned` vs `FileSystem`. So maybe we should also make `audbackend.interface.Unversioned` and ``audbackend.interface.Versioned` available as `Unversioned` and `Versioned`?

Reply:
That was indeed not nice.
I updated the code and now always use `audbackend.backend.FileSystem` in the visible docstring examples.

Reply:
If we like, we could also remove `DoctestFileSystem` and skip the doctests for `date` and `owner`. But I'm also fine with the current solution.